www/js/metrics.js

```diff
@@ -1030,10 +1030,14 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
       return Math.round(v / 1609.34 * 100) / 100;
     }
 
-    $scope.roundCarbon = function(val) {
+    var roundCarbon = function(val) {
```

Comment:
why change `$scope` to `var`?

Reply:
It's called within metrics.js now (not html), no need to be scope var